.circleci/envbuilder.sh

```diff
@@ -161,6 +161,21 @@ installDockerOnRHELViaGCPSSH() {
     local GCP_SSH_KEY_FILE="$1"
     shift
 
+    if [[ "$GCP_IMAGE_FAMILY" == "rhel-8" ]]; then
+        # On RHEL-8 there are some flakey checksum errors when installing packages so
+        # try to reduce the likelihood of these by running makecache before hand.
+        local retryCount=3
+        for _ in $(seq 1 $retryCount); do
+            gcloud compute ssh --ssh-key-file="${GCP_SSH_KEY_FILE}" "$GCP_VM_NAME" --command "sudo yum makecache" \
+                && exitCode=0 && break || exitCode=$? && sleep 5
```

Comment:
Can we turn this into a full on if condition for readabilities sake?

Reply:
Done